### ISSUE TITLE
write_riemann plugin missing in Ubuntu 14.04

### ISSUE THREAD

**Issue by danielkza** [collectd/collectd-ci#22]
/usr/lib/collectd/write_riemann.so doesn't seem to be available after installing Collectd 5.6 in Ubuntu 14.04 (Trusty).

**Comment by archii** [comment]
The `collectd-write_riemann` plugin is not listed in the collectd-ci repos for CentOS 6.  They are not being built on the https://ci.collectd.org Jenkins server.  Is it a matter of missing Riemann requirements on the build hosts?

**Comment by mfournier** [comment]
Yes, in both cases the libriemann-c-client library is missing or too old, in these distros package repositories.

I'm aware the package maintainers made the needed version available for EL7 and debian-jessie backports, which makes building write_riemann possible on these targets. And Ubuntu 16.04 also ships with a recent enough version.

What is left to do is work with whoever is in charge of libriemann-client in the distros you're using, to publish the version collectd requires (>=1.8) in a repository where apt/yum can pull from without any fuss.
